tests/test_jsonyamlxml_writer.py

```diff
@@ -116,3 +121,43 @@ def write_and_parse_snippet(out_format, snippet, temporary_file_path):
 
     return parsed_snippet
 
+
+@pytest.mark.parametrize("out_format", ['yaml', 'xml', 'json'])
+def test_files_without_package(temporary_file_path, out_format):
+    document: Document = minimal_document()
+    document.spdx_id = "SPDXRef-DOCUMENT"
+    file: File = minimal_file()
+    document.add_file(file)
+    describes_relationship: Relationship = Relationship("SPDXRef-DOCUMENT DESCRIBES SPDXRef-File")
+    document.add_relationships(relationship=describes_relationship)
+
+    file_path_with_ending = temporary_file_path + "." + out_format
+    write_anything.write_file(document, file_path_with_ending, validate=False)
+
+    parsed_document: Document = parse_file(file_path_with_ending)[0]
+    parsed_file: File = parsed_document.files[0]
+
+    assert parsed_file.name == "Example File"
+
+
+def minimal_document():
+    document = Document(data_license=License.from_identifier('CC0-1.0'))
+    document.creation_info.set_created_now()
+    return document
+
+
+def minimal_file():
+    file = File(name="Example File", spdx_id="SPDXRef-File", chksum=Algorithm('SHA1', 'SOME-SHA1'))
+    return file
+
+
+@pytest.mark.parametrize("key,value,expected_length",
+                         [("first_key", "second_value", 2), ("first_key", "first_value", 1),
+                          ("second_key", "second_value", 1)])
```

Comment:
nit: `existing_key` and `new_key` would be a bit more descriptive. Similar for values